src/common/type/keyValue.c

```diff
@@ -331,3 +331,39 @@ kvGetList(const KeyValue *const this, const Variant *const key)
 
     FUNCTION_TEST_RETURN(VARIANT_LIST, result);
 }
+
+/**********************************************************************************************************************************/
+KeyValue *
+kvRemove(KeyValue *this, const Variant *key)
+{
+    FUNCTION_TEST_BEGIN();
+        FUNCTION_TEST_PARAM(KEY_VALUE, this);
+        FUNCTION_TEST_PARAM(VARIANT, key);
+    FUNCTION_TEST_END();
+
+    ASSERT(this != NULL);
+    ASSERT(key != NULL);
+
+    MEM_CONTEXT_BEGIN(objMemContext(this))
```

Comment:
Not sure why this mem context switch is needed, since we are not allocating anything?

Reply:
Indeed, fixed in 20d3108